scripts/ansible/kubernetes-cluster/create-vm.yml

```diff
@@ -0,0 +1,113 @@
+---
+- name: Create IBM Cloud VPC VSI
+  hosts: localhost
+  collections:
+   - ibm.cloudcollection
+  tasks:
+    - name: Fetch the variables from var file
+      include_vars:
+        file: vars.yml
+  
+    - name: Configure VPC
+      ibm_is_vpc:
+        name: "{{ name_prefix }}-vpc"
+        state: available
+        id: "{{ vpc.id | default(omit) }}"
+      register: vpc_create_output
+
+    - name: Save VPC as fact
+      set_fact:
+        cacheable: True
+        vpc: "{{ vpc_create_output.resource }}"
+
+    - name: Configure VPC Subnet
+      ibm_is_subnet:
+        name: "{{ name_prefix }}-subnet"
+        state: available
+        id: "{{ subnet.id | default(omit) }}"
+        vpc: "{{ vpc.id }}"
+        total_ipv4_address_count: "{{ total_ipv4_address_count }}"
+        zone: "{{ cluster_zone }}"
+      register: subnet_create_output
+
+    - name: Save VPC Subnet as fact
+      set_fact:
+        cacheable: True
+        subnet: "{{ subnet_create_output.resource }}"
+
+    # NOTE: Uncomment in case we want to create and use a new sshkey 
+    # - name: Configure SSH Key
+    #   ibm_is_ssh_key:
+    #     name: "{{ name_prefix }}-ssh-key"
+    #     public_key: "{{ ssh_public_key }}"
+    #     id: "{{ ssh_key.id | default(omit) }}"
+    #   register: ssh_key_create_output
+
+    # - name: Save SSH Key as fact
+    #   set_fact:
+    #     cacheable: True
+    #     ssh_key: "{{ ssh_key_create_output.resource }}"
+
+    - name: Retrieve image list
+      ibm_is_images_info:
+      register: images_list
+
+    - name: Set VM image name/id dictionary fact
+      set_fact:
+        cacheable: True
+        image_dict: "{{ images_list.resource.images |
+                        items2dict(key_name='name', value_name='id') }}"
+
+    - name: Configure VSI
+      ibm_is_instance:
+        name: "{{ name_prefix }}-windows-vsi"
+        state: available
+        id: "{{ vsi.id | default(omit) }}"
+        vpc: "{{ vpc.id }}"
+        profile: "{{ vsi_profile }}"
+        image: "{{ image_dict[vsi_image] }}"
+        keys:
+          - " {{ sshkey_id }} "         #sshkey id ansible-ssh-key in developer qe group in VPC sshkey
```

Comment:
Please also move the comment to the line defining the specific key ID

Reply:
Done!
Thanks **:)**